pop-up.php

```diff
@@ -0,0 +1,42 @@
+<?php 
```

Comment:
Prefix your page templates with `page` => `page-pop-up.php` What you did now won't break anything but keeping to this convention will make things a lot more structured for you. Future Lelethu will thank you.